en/docs/fragments/travelocity.md

```diff
@@ -1,6 +1,70 @@
 
 {!fragments/deploying-sample-apps.md!}
 
+### Register a service provider
+
+!!! note "Important"
+
+    SAML2 POST Binding requires CORS configurations. Before configuring the service provider, add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml` file to allow HTTP POST requests. 
+
+    ```toml
+    [cors]
+    allow_generic_http_requests = true
+    allow_any_origin = false
+    allowed_origins = [
+        "http://localhost:8080"
+    ]
+    allow_subdomains = false
+    supported_methods = [
+        "GET",
+        "POST",
+        "HEAD",
+        "OPTIONS"
+    ]
+    support_any_header = true
+    supported_headers = []
+    exposed_headers = []
+    supports_credentials = true
+    max_age = 3600
+    tag_requests = false
+    ```
+
+1.  Access the WSO2 IS Management Console (https://<IS_HOST\>:<PORT\>/carbon). 
+
+2.  Navigate to **Main** > **Identity** > **Service Providers** and click **Add**.
+
+3.  Enter **travelocity.com** in the **Service Provider Name** text box,
+    and click **Register**.
```

Comment:
let's use this fragement: https://github.com/wso2/docs-is/blob/new_restructure/en/docs/fragments/register-a-service-provider.md

Reply:
We need sample values to be mentioned in quick starts so cannot use the generic SP fragment.